docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -240,6 +260,22 @@ Release
       admin/publish-artifacts
       admin/publish-docs --production
 
+#. Check that the artifacts are set up correctly:
+
+   .. note:: Ensure that Docker is installed and running, and can be controlled from the current user account.
+      Run ``docker ps`` to check for any problems.
+
+   The following command tests that the client packages can be installed on a number of platforms.
+   This helps to identify any problems with the published artifacts that may not be evident in the regular tests (e.g. S3 permissions or packaging problems).
+
+   .. prompt:: bash (flocker-0.1.2)$
+
+      admin/test-artifacts --flocker-version ${TAG}
+
+   If an error occurs for a single pip installation, create a JIRA issue to investigate it.
```

Comment:
:question: I don't like that this is setting company priorities - say something like "Create an issue and raise it with the team.". It might be sensible at this time to somehow revert to the previous release.